src/methods.jl

```diff
@@ -137,11 +137,11 @@ end
 
 
 # An ifelse node, ifelse is a built-in unfortunately
-#
-cond(_if::Bool, _then, _else) = ifelse(_if, _then, _else)
-function cond(_if::Symbolic{Bool}, _then, _else)
+# So this uses IfElse.jl's ifelse that we imported
+function ifelse(_if::Symbolic{Bool}, _then, _else)
     Term{Union{symtype(_then), symtype(_else)}}(cond, Any[_if, _then, _else])
 end
+Base.@deprecate cond(_if, _then, _else) ifelse(_if, _then, _else)
```

Comment:
The deprecation warning this prints is mildly misleading.
It prints
```julia
┌ Warning: `cond(_if, _then, _else)` is deprecated, use `ifelse(_if, _then, _else)` instead.
```
Because I can't write `IfElse.ifelse` or it will name clash with the rewriter.

Maybe i should hand implement the deprecation warning to force it to say `IfElse.ifelse`

Reply:
I think this is fine... But you're welcome to do the manual thing.